src/main/java/teammates/ui/pagedata/InstructorHomePageData.java

```diff
@@ -45,6 +45,15 @@ public String getRemindParticularStudentsLink() {
         return getInstructorFeedbackRemindParticularStudentsLink(Const.ActionURIs.INSTRUCTOR_HOME_PAGE);
     }
 
+    /**
+     * Retrieves the link to submit the request to email particular students about published session.
```

Comment:
Retrieves the link to submit the request for resending the session published email.